src/tools/idlpp/src/configed.H.in

```diff
@@ -38,7 +38,7 @@
    compiled to language-specific sources, not machine code. */
 
 /* Define the cpu-specific-macro. */
-#define CPU                 "@idlpp_cpu@"
+#define CPU                 ""
```

Comment:
Super nitpick here, but might be nice to set this to `"undefined"` or whatever was used previously if the target cpu wasn't recognized. I also thought about adding something like a `LANGUAGE` definition or something that's set to the language of the generator backend. May or may not be useful... Just a brainfart that I thought could be useful to share(?)